### PR TITLE
Dashboard/events state message

### PR DESCRIPTION
**Pull request by Wardormeur**

### CODE REVIEW

File: src/dashboard/events/cd-dashboard-upcoming-event.vue [hunk 1]
```diff
@@ -2,8 +2,9 @@
   <div class="cd-dashboard-upcoming-event">
     <div v-if="isLoaded" class="cd-dashboard-upcoming-event__content" :class="{ 'cd-dashboard-upcoming-event__content--booked': hasOrder}">
       <div class="cd-dashboard-upcoming-event__main">
-        <h4 v-if="hasOrder">{{ $t('Your next event is "{name}"', { name: event.name }) }}</h4>
-        <h4 v-else>{{ $t('"{name}" is the next session available', { name: event.name }) }}</h4>
+        <h4 v-if="hasOrder">{{ $t('"{name}" is booked', { name: event.name }) }}</h4>
+        <h4 v-else-if="remainingTickets > 0">{{ $t('"{name}" is fully booked', { name: event.name }) }}</h4>
+        <h4 v-else>{{ $t('"{name}" is the available to book', { name: event.name }) }}</h4>
```
Comment (DanielBrierton) [inline src/dashboard/events/cd-dashboard-upcoming-event.vue]: Drop the "the", it's cleaner.

Also, it makes it grammatically correct. 😜

File: src/dashboard/events/cd-dashboard-upcoming-event.vue [hunk 2]
```diff
@@ -2,8 +2,9 @@
   <div class="cd-dashboard-upcoming-event">
     <div v-if="isLoaded" class="cd-dashboard-upcoming-event__content" :class="{ 'cd-dashboard-upcoming-event__content--booked': hasOrder}">
       <div class="cd-dashboard-upcoming-event__main">
-        <h4 v-if="hasOrder">{{ $t('Your next event is "{name}"', { name: event.name }) }}</h4>
-        <h4 v-else>{{ $t('"{name}" is the next session available', { name: event.name }) }}</h4>
+        <h4 v-if="hasOrder">{{ $t('"{name}" is booked', { name: event.name }) }}</h4>
+        <h4 v-else-if="remainingTickets > 0">{{ $t('"{name}" is fully booked', { name: event.name }) }}</h4>
```
Comment (DanielBrierton) [inline src/dashboard/events/cd-dashboard-upcoming-event.vue]: `v-if="remainingTickets <= 0"`

File: src/dashboard/events/cd-dashboard-upcoming-event.vue (hunk 1 above)
Comment (Wardormeur) [reply src/dashboard/events/cd-dashboard-upcoming-event.vue]: Duh oO' My bad

File: src/dashboard/events/cd-dashboard-upcoming-event.vue (hunk 2 above)
Comment (Wardormeur) [reply src/dashboard/events/cd-dashboard-upcoming-event.vue]: Nah, it's booked when there is plenty.
Boolean ftw

**Pull request merged**